### ISSUE TITLE
Accessibility Principles

### ISSUE THREAD

**Issue by abbott567** [dwp/accessibility-manual#14]
Conversation around whether number 2 needs tweaked so that it is clear that accessibility is not always about people using tools. The last sentence should perhaps be re-written to say 'if' or 'whether' they use tools

>**2. Accessible design is good design**
>
>Good design should meet needs and solve problems. If you design something which is inaccessible you will create barriers for people. Good design is not just what looks good, it must be usable. `It must work for everyone, regardless of what tools they use or what impairments they have`.

**Comment by abbott567** [comment]
Principle number 2 updated. Switched content and added 'might' to try and make it a bit clearer. [Git commit for principle number 2 update](https://github.com/dwp/accessibility-manual/commit/1a5b6bca4d2801059369d98790d1614995c35af7).

**Comment by abbott567** [comment]
Principle number 5 was also updated as the phrase 'vulnerable people' may perhaps lead to feelings of sympathy rather than empathy. People are often vulnerable because the environment they have to operate in is not configured for their needs, and this is the message we need to keep pushing. Also, as a department, we provide services for everybody, not just people who may be vulnerable, so we removed the wording to stop it being misconstrued. [Git commit for principle number 5 update](https://github.com/dwp/accessibility-manual/commit/3b0126ac2a043312ea5e8033bb52a37f4d8aada2).
